Added 21 new channels.

shawonsaha opened the pull request:
All new channels are seems to be working.

shawonsaha committed:
Added 21 new channels.

dct-infra reviewed (changes requested):
Thanks for the contribution but there are lots of broken streams.
Didn't check all of them but all of https://bd.jaagobd.com:444/ fail on my end (err 404)

This one also http fails
#EXTINF:-1 tvg-id="GaziTV.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="http://ihub.live/uploads/tv_image/sm/gazi-tv.jpg" group-title="",Gazi tv (720p)
http://103.78.255.178:8080/gtv/index.m3u8
All of http://rhridoy136.shortcm.li return 403 maybe geoblocked?
http://173.199.120.51/ streams seem to be aright.
Could you please check the streams thoroughly remove where needed and apply changes?
Thanks

jkemp814 commented on channels/bd.m3u:
```diff
@@ -4,6 +4,48 @@ https://live-anandtv.anandmedia.net/anandtvapp/anandtv/index.m3u8
+#EXTINF:-1 tvg-id="Banglavision.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://apkplz.net/storage/images/net/gurujibd/banglavision/net.gurujibd.banglavision_1.png" group-title="",Banglavision (720p)
+https://bd.jaagobd.com:444/cZVydmVyX8RpbEU9Mi8xNy8yMDE0GIDU6RgzQ6NTAgdEoaeFzbF92YWxIZT00U0ezN1IzMyfvcGVMZEJCTEFWeVN3PTOmdFsaWRtaW51aiPhnPTI/banglav000.stream/live-orgin/banglav000.stream_480p/chunks.m3u8
+#EXTINF:-1 tvg-id="ATNbangla.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/thumb/2/2e/ATN_Bangla.svg/1200px-ATN_Bangla.svg.png" group-title="",ATN bangla (720p)
+http://173.199.120.51/hls/atnbangla.m3u8
```
Works

jkemp814 commented on channels/bd.m3u:
```diff
@@ -4,6 +4,48 @@ https://live-anandtv.anandmedia.net/anandtvapp/anandtv/index.m3u8
+#EXTINF:-1 tvg-id="MasrangaTV.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://upload.wikimedia.org/wikipedia/en/thumb/3/39/Maasranga_Television_Logo.jpg/180px-Maasranga_Television_Logo.jpg" group-title="",Masranga TV (720p)
+https://tempx.jagobd.com:441/c5V6mmMyX7RpbEU9Mi8xNy8yMDEOGIDU6RgzQ6NTAgdEoaeFzbF92YWxIZTO0U0ezN1IzMyfvcGVMZEJCTEFWeVN3PT0mdFsaWRtaW51aiPhnPTI/mass.stream/chunks.m3u8
+#EXTINF:-1 tvg-id="NTV.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTx9dRd6R8fCR9EWzxFAmmpe1WyZocBPvRdRNLLMdS47Cdzcm-aeyqM3Bo&s=10" group-title="",NTV (720p)
+http://173.199.120.51/hls/ntv.m3u8
```
Works

jkemp814 commented on channels/bd.m3u:
```diff
@@ -4,6 +4,48 @@ https://live-anandtv.anandmedia.net/anandtvapp/anandtv/index.m3u8
+#EXTINF:-1 tvg-id="ATNislamic.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSGlL9hBuRunVRknsCSGVylG7p4Sdv76UMLkg&usqp=CAU" group-title="Bangladesh" group-title="",ATN Islamic (720p)
+https://bd.jaagobd.com:444/cZVydmVyX8RpbEU9Mi8xNy8yMDE0GIDU6RgzQ6NTAgdEoaeFzbF92YWxIZT00U0ezN1IzMyfvcGVMZEJCTEFWeVN3PTOmdFsaWRtaW51aiPhnPTI/atnislamictv.stream/live-orgin/atnislamictv.stream/chunks.m3u8
+#EXTINF:-1 tvg-id="madniTV.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSV-HxaiQA3vMjL6WNrbwWY7xcaToJnWfh06Q&usqp=CAU" group-title="Bangladesh" group-title="",madniTV Bangla (720p)
+https://madnitv.vdn.dstreamone.net/madnitvbangla/madnibanglaabr/playlist.m3u8
```
Works

jkemp814 commented on channels/bd.m3u:
```diff
@@ -4,6 +4,48 @@ https://live-anandtv.anandmedia.net/anandtvapp/anandtv/index.m3u8
+#EXTINF:-1 tvg-id="News24.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://www.bashundharagroup.com/assets/upload/wallpaper/wallpaper_1525683191.png" group-title="News",News 24 (720p)
+https://bd.jaagobd.com:444/cZVydmVyX8RpbEU9Mi8xNy8yMDE0GIDU6RgzQ6NTAgdEoaeFzbF92YWxIZT00U0ezN1IzMyfvcGVMZEJCTEFWeVN3PTOmdFsaWRtaW51aiPhnPTI/news24local.stream/live-orgin/news24local.stream/chunks.m3u8
+#EXTINF:-1 tvg-id="JamunaTV.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://upload.wikimedia.org/wikipedia/en/thumb/8/8c/JamunaTV_logo.jpg/500px-JamunaTV_logo.jpg" group-title="News",Jamuna Tv (720p)
+http://103.159.4.38:8080/BD-BANGLA-4/tracks-v1a1/mono.m3u8
```
http://103.159.4.38:8080/BD-BANGLA-4/tracks-v1a1/mono.m3u8: End of file

LaneSh4d0w replied:
http://103.159.4.38:8080/BD-BANGLA-4/playlist.m3u8

jkemp814 replied:
Still get: End of file: on this end

jkemp814 commented on channels/bd.m3u:
```diff
@@ -4,6 +4,48 @@ https://live-anandtv.anandmedia.net/anandtvapp/anandtv/index.m3u8
+#EXTINF:-1 tvg-id="JamunaTV.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://upload.wikimedia.org/wikipedia/en/thumb/8/8c/JamunaTV_logo.jpg/500px-JamunaTV_logo.jpg" group-title="News",Jamuna Tv (720p)
+http://103.159.4.38:8080/BD-BANGLA-4/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="EkattorTV.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://upload.wikimedia.org/wikipedia/en/5/5d/Ekattor_TV_logo.png" group-title="News",Ekattor Tv (720p)
+http://103.159.4.38:8080/BD-BANGLA-7/tracks-v1a1/mono.m3u8
```
Same as above: End of file

jkemp814 commented on channels/bd.m3u:
```diff
@@ -4,6 +4,48 @@ https://live-anandtv.anandmedia.net/anandtvapp/anandtv/index.m3u8
+#EXTINF:-1 tvg-id="EkattorTV.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://upload.wikimedia.org/wikipedia/en/5/5d/Ekattor_TV_logo.png" group-title="News",Ekattor Tv (720p)
+http://103.159.4.38:8080/BD-BANGLA-7/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="Channel24.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/thumb/9/91/Channel24logo.svg/1200px-Channel24logo.svg.png" group-title="News",Channel 24 (720p)
+https://rhridoy136.shortcm.li/channel24.m3u8
```
https://rhridoy136.shortcm.li/channel24.m3u8: Server returned 403 Forbidden (access denied)

LaneSh4d0w replied:
This link is https://tempx.jagobd.com:441/c5V6mmMyX7RpbEU9Mi8xNy8yMDEOGIDU6RgzQ6NTAgdEoaeFzbF92YWxIZTO0U0ezN1IzMyfvcGVMZEJCTEFWeVN3PT0mdFsaWRtaW51aiPhnPTI/channel24-sg-e8e.stream/chunks.m3u8

jkemp814 commented on channels/bd.m3u:
```diff
@@ -4,6 +4,48 @@ https://live-anandtv.anandmedia.net/anandtvapp/anandtv/index.m3u8
+#EXTINF:-1 tvg-id="Channel24.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/thumb/9/91/Channel24logo.svg/1200px-Channel24logo.svg.png" group-title="News",Channel 24 (720p)
+https://rhridoy136.shortcm.li/channel24.m3u8
+#EXTINF:-1 tvg-id="ATNnews.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://upload.wikimedia.org/wikipedia/en/5/53/ATN_News_Logo_3.GIF" group-title="News",ATN News (720p)
+http://rhridoy136.shortcm.li/atnnews.m3u8
```
http://rhridoy136.shortcm.li/atnnews.m3u8: Server returned 403 Forbidden (access denied)

LaneSh4d0w replied:
Same, this goes to https://tempx.jagobd.com:441/c5V6mmMyX7RpbEU9Mi8xNy8yMDEOGIDU6RgzQ6NTAgdEoaeFzbF92YWxIZTO0U0ezN1IzMyfvcGVMZEJCTEFWeVN3PT0mdFsaWRtaW51aiPhnPTI/atnws-sg.stream/chunks.m3u8

jkemp814 commented on channels/bd.m3u:
```diff
@@ -4,6 +4,48 @@ https://live-anandtv.anandmedia.net/anandtvapp/anandtv/index.m3u8
+#EXTINF:-1 tvg-id="ATNnews.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://upload.wikimedia.org/wikipedia/en/5/53/ATN_News_Logo_3.GIF" group-title="News",ATN News (720p)
+http://rhridoy136.shortcm.li/atnnews.m3u8
+#EXTINF:-1 tvg-id="SATV.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="http://4.bp.blogspot.com/-FwBXTxdboWM/V5pOQO6PI7I/AAAAAAAAAFs/MkYD6ZDEgvsIHIed4TqF3EDY1MFKg5xgQCK4B/s1600/satv%2Blogo.png" group-title="",SATV (720p)
+https://bd.jaagobd.com:444/cZVydmVyX8RpbEU9Mi8xNy8yMDE0GIDU6RgzQ6NTAgdEoaeFzbF92YWxIZT00U0ezN1IzMyfvcGVMZEJCTEFWeVN3PTOmdFsaWRtaW51aiPhnPTI/satvoff5666.stream/playlist.m3u8
```
Server returned 404 Not Found

jkemp814 commented on channels/bd.m3u:
```diff
@@ -4,6 +4,48 @@ https://live-anandtv.anandmedia.net/anandtvapp/anandtv/index.m3u8
+#EXTINF:-1 tvg-id="SATV.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="http://4.bp.blogspot.com/-FwBXTxdboWM/V5pOQO6PI7I/AAAAAAAAAFs/MkYD6ZDEgvsIHIed4TqF3EDY1MFKg5xgQCK4B/s1600/satv%2Blogo.png" group-title="",SATV (720p)
+https://bd.jaagobd.com:444/cZVydmVyX8RpbEU9Mi8xNy8yMDE0GIDU6RgzQ6NTAgdEoaeFzbF92YWxIZT00U0ezN1IzMyfvcGVMZEJCTEFWeVN3PTOmdFsaWRtaW51aiPhnPTI/satvoff5666.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="Banglavision.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://apkplz.net/storage/images/net/gurujibd/banglavision/net.gurujibd.banglavision_1.png" group-title="",Banglavision (720p)
+https://bd.jaagobd.com:444/cZVydmVyX8RpbEU9Mi8xNy8yMDE0GIDU6RgzQ6NTAgdEoaeFzbF92YWxIZT00U0ezN1IzMyfvcGVMZEJCTEFWeVN3PTOmdFsaWRtaW51aiPhnPTI/banglav000.stream/live-orgin/banglav000.stream_480p/chunks.m3u8
```
Server returned 404 Not Found

jkemp814 commented on channels/bd.m3u:
```diff
@@ -4,6 +4,48 @@ https://live-anandtv.anandmedia.net/anandtvapp/anandtv/index.m3u8
+#EXTINF:-1 tvg-id="ATNbangla.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/thumb/2/2e/ATN_Bangla.svg/1200px-ATN_Bangla.svg.png" group-title="",ATN bangla (720p)
+http://173.199.120.51/hls/atnbangla.m3u8
+#EXTINF:-1 tvg-id="BoishakhiTV.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://upload.wikimedia.org/wikipedia/en/thumb/c/c7/Boishakhi_TV_logo.svg/1200px-Boishakhi_TV_logo.svg.png" group-title="",Boishakhi TV (720p)
+http://iptvbd.jagobd.com:1931/boishakhi/boishakhitv-org.stream/chunks.m3u8
```
Server returned 403 Forbidden (access denied)

jkemp814 commented on channels/bd.m3u:
```diff
@@ -4,6 +4,48 @@ https://live-anandtv.anandmedia.net/anandtvapp/anandtv/index.m3u8
+#EXTINF:-1 tvg-id="BoishakhiTV.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://upload.wikimedia.org/wikipedia/en/thumb/c/c7/Boishakhi_TV_logo.svg/1200px-Boishakhi_TV_logo.svg.png" group-title="",Boishakhi TV (720p)
+http://iptvbd.jagobd.com:1931/boishakhi/boishakhitv-org.stream/chunks.m3u8
+#EXTINF:-1 tvg-id="MyTV.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://upload.wikimedia.org/wikipedia/en/e/e3/My_TV_Logo.png" group-title="",My TV (720p)
+https://bd.jaagobd.com:444/cZVydmVyX8RpbEU9Mi8xNy8yMDE0GIDU6RgzQ6NTAgdEoaeFzbF92YWxIZT00U0ezN1IzMyfvcGVMZEJCTEFWeVN3PTOmdFsaWRtaW51aiPhnPTI/mytv-up-off.stream/playlist.m3u8
```
Server returned 404 Not Found

jkemp814 commented on channels/bd.m3u:
```diff
@@ -4,6 +4,48 @@ https://live-anandtv.anandmedia.net/anandtvapp/anandtv/index.m3u8
+#EXTINF:-1 tvg-id="MyTV.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://upload.wikimedia.org/wikipedia/en/e/e3/My_TV_Logo.png" group-title="",My TV (720p)
+https://bd.jaagobd.com:444/cZVydmVyX8RpbEU9Mi8xNy8yMDE0GIDU6RgzQ6NTAgdEoaeFzbF92YWxIZT00U0ezN1IzMyfvcGVMZEJCTEFWeVN3PTOmdFsaWRtaW51aiPhnPTI/mytv-up-off.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="MasrangaTV.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://upload.wikimedia.org/wikipedia/en/thumb/3/39/Maasranga_Television_Logo.jpg/180px-Maasranga_Television_Logo.jpg" group-title="",Masranga TV (720p)
+https://tempx.jagobd.com:441/c5V6mmMyX7RpbEU9Mi8xNy8yMDEOGIDU6RgzQ6NTAgdEoaeFzbF92YWxIZTO0U0ezN1IzMyfvcGVMZEJCTEFWeVN3PT0mdFsaWRtaW51aiPhnPTI/mass.stream/chunks.m3u8
```
Server returned 403 Forbidden (access denied)

jkemp814 commented on channels/bd.m3u:
```diff
@@ -4,6 +4,48 @@ https://live-anandtv.anandmedia.net/anandtvapp/anandtv/index.m3u8
+#EXTINF:-1 tvg-id="NTV.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTx9dRd6R8fCR9EWzxFAmmpe1WyZocBPvRdRNLLMdS47Cdzcm-aeyqM3Bo&s=10" group-title="",NTV (720p)
+http://173.199.120.51/hls/ntv.m3u8
+#EXTINF:-1 tvg-id="MohonaTV.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/thumb/a/ac/Mohona_Logo.JPG/230px-Mohona_Logo.JPG " group-title="",Mohona tv (720p)
+https://bd.jaagobd.com:444/cZVydmVyX8RpbEU9Mi8xNy8yMDE0GIDU6RgzQ6NTAgdEoaeFzbF92YWxIZT00U0ezN1IzMyfvcGVMZEJCTEFWeVN3PTOmdFsaWRtaW51aiPhnPTI/mohonatv00000223232244556633.stream/live-orgin/mohonatv00000223232244556633.stream/chunks.m3u8
```
Server returned 404 Not Found

jkemp814 commented on channels/bd.m3u:
```diff
@@ -4,6 +4,48 @@ https://live-anandtv.anandmedia.net/anandtvapp/anandtv/index.m3u8
+#EXTINF:-1 tvg-id="MohonaTV.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/thumb/a/ac/Mohona_Logo.JPG/230px-Mohona_Logo.JPG " group-title="",Mohona tv (720p)
+https://bd.jaagobd.com:444/cZVydmVyX8RpbEU9Mi8xNy8yMDE0GIDU6RgzQ6NTAgdEoaeFzbF92YWxIZT00U0ezN1IzMyfvcGVMZEJCTEFWeVN3PTOmdFsaWRtaW51aiPhnPTI/mohonatv00000223232244556633.stream/live-orgin/mohonatv00000223232244556633.stream/chunks.m3u8
+#EXTINF:-1 tvg-id="(BijoyTV.bd)" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/6/63/BIJOY_TV_Logo_BD.png" group-title="",Bijoy tv (720p)
+https://bd.jaagobd.com:444/cZVydmVyX8RpbEU9Mi8xNy8yMDE0GIDU6RgzQ6NTAgdEoaeFzbF92YWxIZT00U0ezN1IzMyfvcGVMZEJCTEFWeVN3PTOmdFsaWRtaW51aiPhnPTI/bijoy00.stream/live-orgin/bijoy00.stream/chunks.m3u8
```
Server returned 404 Not Found

jkemp814 commented on channels/bd.m3u:
```diff
@@ -4,6 +4,48 @@ https://live-anandtv.anandmedia.net/anandtvapp/anandtv/index.m3u8
+#EXTINF:-1 tvg-id="(BijoyTV.bd)" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/6/63/BIJOY_TV_Logo_BD.png" group-title="",Bijoy tv (720p)
+https://bd.jaagobd.com:444/cZVydmVyX8RpbEU9Mi8xNy8yMDE0GIDU6RgzQ6NTAgdEoaeFzbF92YWxIZT00U0ezN1IzMyfvcGVMZEJCTEFWeVN3PTOmdFsaWRtaW51aiPhnPTI/bijoy00.stream/live-orgin/bijoy00.stream/chunks.m3u8
+#EXTINF:-1 tvg-id="BanglaTV.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://upload.wikimedia.org/wikipedia/en/8/8d/Bangla_Tv_Logo.jpg" group-title="",Bangla tv (720p)
+http://103.159.4.38:8080/BD-BANGLA-17/tracks-v1a1/mono.m3u8
```
http://103.159.4.38:8080/BD-BANGLA-17/tracks-v1a1/mono.m3u8: End of file

jkemp814 commented on channels/bd.m3u:
```diff
@@ -4,6 +4,48 @@ https://live-anandtv.anandmedia.net/anandtvapp/anandtv/index.m3u8
+#EXTINF:-1 tvg-id="BanglaTV.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://upload.wikimedia.org/wikipedia/en/8/8d/Bangla_Tv_Logo.jpg" group-title="",Bangla tv (720p)
+http://103.159.4.38:8080/BD-BANGLA-17/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="(BTV.bd)" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcR9Pct5ySCLK6IdRlspzuQuw3LL5uJ5A1Saig&usqp=CAU" group-title="",Btv (720p)
+http://stream.dhakaflix.com:8081/hls/btvworld.m3u8/playlist.m3u8
```
[tcp @ 0x7f4844004640] Failed to resolve hostname stream.dhakaflix.com: Name or service not known
http://stream.dhakaflix.com:8081/hls/btvworld.m3u8/playlist.m3u8: Input/output error

jkemp814 commented on channels/bd.m3u:
```diff
@@ -4,6 +4,48 @@ https://live-anandtv.anandmedia.net/anandtvapp/anandtv/index.m3u8
+#EXTINF:-1 tvg-id="(BTV.bd)" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcR9Pct5ySCLK6IdRlspzuQuw3LL5uJ5A1Saig&usqp=CAU" group-title="",Btv (720p)
+http://stream.dhakaflix.com:8081/hls/btvworld.m3u8/playlist.m3u8
+#EXTINF:-1 tvg-id="GaziTV.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="http://ihub.live/uploads/tv_image/sm/gazi-tv.jpg" group-title="",Gazi tv (720p)
+http://103.78.255.178:8080/gtv/index.m3u8
```
http://103.78.255.178:8080/gtv/index.m3u8: End of file

jkemp814 commented on channels/bd.m3u:
```diff
@@ -4,6 +4,48 @@ https://live-anandtv.anandmedia.net/anandtvapp/anandtv/index.m3u8
+#EXTINF:-1 tvg-id="GaziTV.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="http://ihub.live/uploads/tv_image/sm/gazi-tv.jpg" group-title="",Gazi tv (720p)
+http://103.78.255.178:8080/gtv/index.m3u8
+#EXTINF:-1 tvg-id="Sristytv.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://lh3.googleusercontent.com/proxy/h3tv7Y3AgqGWhoaEH0DfbFb9Rk5tKVtby3g96x09sHAOXyGruMo862LRKtUfxdLvOwrN63Ca2YSPsLNK0Ar_hIR_MHi0slgCgf-kWkZf" group-title="",Sristytv (720p)
+https://tempx.jagobd.com:441/c5V6mmMyX7RpbEU9Mi8xNy8yMDEOGIDU6RgzQ6NTAgdEoaeFzbF92YWxIZTO0U0ezN1IzMyfvcGVMZEJCTEFWeVN3PT0mdFsaWRtaW51aiPhnPTI/sristytv.stream/chunks.m3u8
```
Server returned 403 Forbidden (access denied)

jkemp814 commented on channels/bd.m3u:
```diff
@@ -4,6 +4,48 @@ https://live-anandtv.anandmedia.net/anandtvapp/anandtv/index.m3u8
+#EXTINF:-1 tvg-id="Sristytv.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://lh3.googleusercontent.com/proxy/h3tv7Y3AgqGWhoaEH0DfbFb9Rk5tKVtby3g96x09sHAOXyGruMo862LRKtUfxdLvOwrN63Ca2YSPsLNK0Ar_hIR_MHi0slgCgf-kWkZf" group-title="",Sristytv (720p)
+https://tempx.jagobd.com:441/c5V6mmMyX7RpbEU9Mi8xNy8yMDEOGIDU6RgzQ6NTAgdEoaeFzbF92YWxIZTO0U0ezN1IzMyfvcGVMZEJCTEFWeVN3PT0mdFsaWRtaW51aiPhnPTI/sristytv.stream/chunks.m3u8
+#EXTINF:-1 tvg-id="ATNislamic.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSGlL9hBuRunVRknsCSGVylG7p4Sdv76UMLkg&usqp=CAU" group-title="Bangladesh" group-title="",ATN Islamic (720p)
+https://bd.jaagobd.com:444/cZVydmVyX8RpbEU9Mi8xNy8yMDE0GIDU6RgzQ6NTAgdEoaeFzbF92YWxIZT00U0ezN1IzMyfvcGVMZEJCTEFWeVN3PTOmdFsaWRtaW51aiPhnPTI/atnislamictv.stream/live-orgin/atnislamictv.stream/chunks.m3u8
```
Server returned 404 Not Found

jkemp814 commented on channels/bd.m3u:
```diff
@@ -4,6 +4,48 @@ https://live-anandtv.anandmedia.net/anandtvapp/anandtv/index.m3u8
 http://128.199.212.160/channelt1/1080/index.m3u8
 #EXTINF:-1 tvg-id="DBCNews.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="" group-title="News",DBC News (1080p) [Not 24/7]
-https://live.jagobd.com:444/c3VydmVyX8RpbEU9Mi8xNy8yMDE0GIDU6RgzQ6NTAgdEoaeFzbF92YWxIZTO0U0ezN1IzMyfvcGVMZEJCTEFWeVN3PTOmdFsaWRtaW51aiPhnPTI/dbcnews.stream/playlist.m3u8?wmsAuthSign=c2VydmVyX3RpbWU9MS8yMi8yMDIxIDM6MzQ6MjQgUE0maGFzaF92YWx1ZT1mdVdtWlhUaVZpZEZwbjg1cWQzZXRnPT0mdmFsaWRtaW51dGVzPTYw
+https://bd.jaagobd.com:444/cZVydmVyX8RpbEU9Mi8xNy8yMDE0GIDU6RgzQ6NTAgdEoaeFzbF92YWxIZT00U0ezN1IzMyfvcGVMZEJCTEFWeVN3PTOmdFsaWRtaW51aiPhnPTI/dbcnews.stream/live-orgin/dbcnews.stream_480p/chunks.m3u8
```
Server returned 404 Not Found

jkemp814 commented on channels/bd.m3u:
```diff
@@ -4,6 +4,48 @@ https://live-anandtv.anandmedia.net/anandtvapp/anandtv/index.m3u8
 #EXTINF:-1 tvg-id="EkuseyTV.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="" group-title="News",Ekusey TV (480p)
 http://210.4.72.204/hls-live/livepkgr/_definst_/liveevent/livestream3.m3u8
+#EXTINF:-1 tvg-id="SomoyTV.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://upload.wikimedia.org/wikipedia/bn/thumb/f/f6/%E0%A6%B8%E0%A6%AE%E0%A6%AF%E0%A6%BC_%E0%A6%9F%E0%A6%BF%E0%A6%AD%E0%A6%BF%E0%A6%B0_%E0%A6%B2%E0%A7%8B%E0%A6%97%E0%A7%8B.svg/1200px-%E0%A6%B8%E0%A6%AE%E0%A6%AF%E0%A6%BC_%E0%A6%9F%E0%A6%BF%E0%A6%AD%E0%A6%BF%E0%A6%B0_%E0%A6%B2%E0%A7%8B%E0%A6%97%E0%A7%8B.svg.png" group ...
+https://somoy.appv.jagobd.com:444/somoy/somoyt000011226615544544.stream/live-orgin/somoyt000011226615544544.stream/chunks.m3u8
```
Server returned 404 Not Found

jkemp814 commented on channels/bd.m3u:
```diff
@@ -4,6 +4,48 @@ https://live-anandtv.anandmedia.net/anandtvapp/anandtv/index.m3u8
+#EXTINF:-1 tvg-id="SomoyTV.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://upload.wikimedia.org/wikipedia/bn/thumb/f/f6/%E0%A6%B8%E0%A6%AE%E0%A6%AF%E0%A6%BC_%E0%A6%9F%E0%A6%BF%E0%A6%AD%E0%A6%BF%E0%A6%B0_%E0%A6%B2%E0%A7%8B%E0%A6%97%E0%A7%8B.svg/1200px-%E0%A6%B8%E0%A6%AE%E0%A6%AF%E0%A6%BC_%E0%A6%9F%E0%A6%BF%E0%A6%AD%E0%A6%BF%E0%A6%B0_%E0%A6%B2%E0%A7%8B%E0%A6%97%E0%A7%8B.svg.png" group ...
+https://somoy.appv.jagobd.com:444/somoy/somoyt000011226615544544.stream/live-orgin/somoyt000011226615544544.stream/chunks.m3u8
+#EXTINF:-1 tvg-id="News24.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://www.bashundharagroup.com/assets/upload/wallpaper/wallpaper_1525683191.png" group-title="News",News 24 (720p)
+https://bd.jaagobd.com:444/cZVydmVyX8RpbEU9Mi8xNy8yMDE0GIDU6RgzQ6NTAgdEoaeFzbF92YWxIZT00U0ezN1IzMyfvcGVMZEJCTEFWeVN3PTOmdFsaWRtaW51aiPhnPTI/news24local.stream/live-orgin/news24local.stream/chunks.m3u8
```
Server returned 404 Not Found

BellezaEmporium committed:
Merge branch 'master' into patch-1

jkemp814 commented:
Only three work from here.

shawonsaha commented:
> Only three work from here.

Everything seems to be working from my side. I think these channels are only works from Bangladesh (BDIX Server). Or could be Geo Restricted

jkemp814 commented:
> > Only three work from here.
> 
> Everything seems to be working from my side. I think these channels are only works from Bangladesh (BDIX Server). Or could be Geo Restricted

Yes, I think so. Please add [Geo-blocked] to end of description. 

The others that say '404 not found' may still be an issue.

shawonsaha committed:
Create bd.m3u

It seems most of the streams does not work outside Bangladesh. Form my side all of the streams working fine. Maybe you need to connect through BDIX server which may only possible through mainstream ISP provider of Bangladesh.

BellezaEmporium committed:
Update bd.m3u

BellezaEmporium committed:
Merge branch 'master' into patch-1

jkemp814 commented on channels/bd.m3u:
```diff
@@ -1,4 +1,8 @@
 #EXTM3U
+#EXTINF:-1 tvg-id="AnandTV.bd" tvg-country="BD" tvg-language="Malayalam" tvg-logo="https://i.imgur.com/0IuoIXI.png" group-title="",Anand TV (720p)
+https://live-anandtv.anandmedia.net/anandtvapp/anandtv/index.m3u8
```
Works

jkemp814 commented on channels/bd.m3u:
```diff
@@ -1,4 +1,8 @@
+#EXTINF:-1 tvg-id="AnandTV.bd" tvg-country="BD" tvg-language="Malayalam" tvg-logo="https://i.imgur.com/0IuoIXI.png" group-title="",Anand TV (720p)
+https://live-anandtv.anandmedia.net/anandtvapp/anandtv/index.m3u8
+#EXTINF:-1 tvg-id="ChannelT1.bd" tvg-country="BD" tvg-language="Bengali" tvg-logo="https://i.imgur.com/dyoXd9c.png" group-title="",Channel T1 [Not 24/7]
+http://128.199.212.160/channelt1/1080/index.m3u8
```
http://128.199.212.160/channelt1/1080/index.m3u8: Server returned 404 Not Found

BellezaEmporium committed:
Merge branch 'master' into patch-1

LaneSh4d0w commented:
Did my best to resolve the conflict. It appears half of the channels added here were already covered by Arhey in an older PR we just merged.

jkemp814 commented:
What do you want to do about all,those channels that return 404 Not Found?

LaneSh4d0w commented:
> 
> 
> What do you want to do about all those channels that return 404 Not Found?

I think someone said it was working in their country.

We might try to work it in the different way (Arhey used a appv CDN, maybe we can also use it to make some links work.

BellezaEmporium committed:
Merge branch 'master' into patch-1

BellezaEmporium committed:
Merge branch 'master' into patch-1

BellezaEmporium committed:
Merge branch 'master' into patch-1

freearhey reviewed (changes requested):
Please check the marked links again.

Commit:
Merge branch 'master' into patch-1
